src/components/patients/patientTable/index.tsx

```diff
@@ -1,90 +1,70 @@
 import React from "react";
-import { Link } from "react-router-dom";
-import { Cell, Column, Table } from "@blueprintjs/table";
-import { Icon, H3 } from "@blueprintjs/core";
-import { ROUTE_PATIENT } from "../../../constants";
-import { getPatients, getCount } from "services/api";
-import { Patient } from "types";
+import { Icon, H3, Button } from "@blueprintjs/core";
+import { PatientBundle } from "types";
+import PatientCardTable from "components/patients/patientTable/patientCardTable";
+import { PATIENT_SHOWN } from "../../../constants";
+import { requestNextPatients } from "services/api";
 
 import "./style.css";
 
-const PatientTable = () => {
-  const [patients, setPatients] = React.useState([] as Patient[]);
-  const [patientCount, setPatientCount] = React.useState("");
+interface Props {
+  bundle: PatientBundle;
+}
 
-  const renderPatientAttribute = (
-    attribute: "id" | "identifier" | "firstName" | "lastName" | "age",
-    index: number
-  ) => (
-    <Cell>
-      <React.Fragment>
-        <Link to={`${ROUTE_PATIENT}/${patients[index].id}`}>
-          {patients[index][attribute] || "unknown"}
-        </Link>
-      </React.Fragment>
-    </Cell>
-  );
+const PatientTable = ({ bundle }: Props) => {
+  const [pageIndex, setPageIndex] = React.useState(0);
+  const [leftDisabled, setLeftDisabled] = React.useState(true);
+  const [rightDisabled, setRightDisabled] = React.useState(false);
+  const [patientBundle, setPatientBundle] = React.useState(bundle);
```

Comment:
I'm not a React expert but I find this weird that you have `bundle` both passed from a parent component and as a state.

Reply:
removed